forage-android/src/main/java/com/joinforage/forage/android/core/services/launchdarkly/LDManager.kt

```diff
@@ -22,14 +23,24 @@ internal object LDContextKind {
     const val SERVICE = "service"
 }
 
+// vault-primary-traffic-percentage
 internal val ALWAYS_BT_PERCENT = 100.0
 internal val ALWAYS_VGS_PERCENT = 0.0
 
+// rosetta-traffic-percentage
+internal val ALWAYS_ROSETTA_PERCENT = 100.0
+internal val ALWAYS_THIRD_PARTY_PERCENT = 0.0
+
 internal fun computeVaultType(trafficPrimaryPercentFlag: Double): VaultType {
     val randomNum = Math.random() * 100
     return if (randomNum < trafficPrimaryPercentFlag) VaultType.BT_VAULT_TYPE else VaultType.VGS_VAULT_TYPE
 }
```

Comment:
can drop `vault-primary-traffic-percentage` entirely in favor of `rosetta-traffic-percentage`, as discussed in slack

Reply:
[53a8f93](https://github.com/teamforage/forage-android-sdk/pull/258/commits/53a8f93c000e0c0582d10934d09a5f8b4d58354c)